contents/docs/experiments/analyzing-results.mdx

```diff
@@ -0,0 +1,163 @@
+  free: none
+  selfServe: full
+  enterprise: full
+---
+
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const ExposuresPanelLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/exposures_light_04d3ee6606.png"
+export const ExposuresPanelDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/exposures_dark_6cf47da9eb.png"
+export const DeltaChartLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/delta_chart_light_3010641eff.png"
+export const DeltaChartDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/delta_chart_dark_82a4470a98.png"
+export const DeltaChartHoverLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_light_8fe379428f.png"
+export const DeltaChartHoverDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/popup_dark_cd122710e7.png"
+export const FunnelBreakdownLight = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_light_e933e07d24.png"
+export const FunnelBreakdownDark = "https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/funnel_dark_c6a7c3e830.png"
+
+Once your experiment has been running and gathering data, you'll want to analyze the results to determine if your changes had a statistically significant impact. This guide walks you through understanding and interpreting experiment results in PostHog.
+
+## Understanding the results page
+
+### Exposures panel
+
+The exposures panel shows you how many users have been exposed to each variant of your experiment:
+
+<ProductScreenshot
+    imageLight={ExposuresPanelLight} 
+    imageDark={ExposuresPanelDark}
+    alt="Experiment exposures panel showing user distribution across variants" 
+    classes="rounded"
+/>
+
+This collapsible panel displays:
+- **Exposure count**: The total number of unique users who have seen each variant
+- **Percentage distribution**: How users are split between variants (typically 50/50 for two variants)
+
+The exposure numbers help you verify that:
+1. Your experiment is running and users are being assigned to variants
+2. The split between variants matches your expected allocation
+3. You have sufficient sample size for meaningful results
+
+### Metric results and significance
+
+For each metric you're tracking, PostHog displays a delta chart showing the performance of each variant compared to the control:
+
+<ProductScreenshot
+    imageLight={DeltaChartLight} 
+    imageDark={DeltaChartDark}
+    alt="Delta chart showing experiment results with confidence intervals" 
+    classes="rounded"
+/>
+
+## How to determine if a variant is significant
+
+The most important question when analyzing results is: **"Is my variant statistically significant?"**
+
+Look for these clear visual indicators in the delta chart:
+
+1. **Color-coded deltas**: When a variant achieves statistical significance:
+   - **Green with ↑ arrow**: The variant is winning (positive significant change)
```

Comment:
Does this language still make sense given that experiments support goals for numbers to go ⬆️  and ⬇️  now?